api/src/test/kotlin/com/ford/internalprojects/peoplemover/report/ReportGeneratorControllerTest.kt

```diff
@@ -167,31 +171,61 @@ class ReportGeneratorControllerTest {
     @Test
     fun `GET should return 400 with invalid space name`() {
         mockMvc.perform(get("$basePeopleReportsUrl?spaceUuid=fakeSpace&requestedDate=${mar1}")
-            .header("Authorization", "Bearer GOOD_TOKEN"))
-            .andExpect(status().isBadRequest)
+                .header("Authorization", "Bearer GOOD_TOKEN"))
+                .andExpect(status().isBadRequest)
     }
 
     @Test
     fun `GET should return all space names, who created the space, and all users related to that space if users is authorized`() {
         val result = mockMvc
-            .perform(
-                get(baseSpaceReportsUrl)
-                    .header("Authorization", "Bearer GOOD_TOKEN")
-            )
-            .andExpect(status().isOk)
-            .andReturn()
+                .perform(
+                        get(baseSpaceReportsUrl)
+                                .header("Authorization", "Bearer GOOD_TOKEN")
+                )
+                .andExpect(status().isOk)
+                .andReturn()
 
         val actualSpaceReport = objectMapper.readValue<List<SpaceReportItem>>(
-            result.response.contentAsString,
-            objectMapper
-                .typeFactory
-                .constructCollectionType(MutableList::class.java, SpaceReportItem::class.java)
+                result.response.contentAsString,
+                objectMapper
+                        .typeFactory
+                        .constructCollectionType(MutableList::class.java, SpaceReportItem::class.java)
```

Comment:
Inconsistent spacing?

Reply:
Those are different objects. We could probably pull that out though

Reply:
I was just noting that on line 189 and 190 is like 2 spaces vs 191 and 192 is 4 it looks like..